docs/src/components/DocsPageBuilder/BpkMarkdownRenderer.js

```diff
@@ -52,6 +56,33 @@ const BpkMarkdownRenderer = (props: Props) => {
 
   const renderers = {};
 
+  renderers.paragraph = Paragraph;
+  renderers.link = BpkLink;
+  renderers.blockquote = BpkBlockquote;
+  renderers.tableRow = BpkTableRow;
+  renderers.tableHead = BpkTableHead;
+  renderers.tableBody = BpkTableBody;
+
+  renderers.tableCell = tableCellProps => {
+    const { isHeader, children, ...tableCellRest } = tableCellProps;
+    const CellComponent = isHeader ? BpkTableHeadCell : BpkTableCell;
+    return <CellComponent {...tableCellRest} />;
+  };
+
+  renderers.image = imageProps => {
+    const { alt, src, ...imageRest } = imageProps;
+    return (
+      // Can't use BpkImage here because it requires us to know width and height.
+      /* eslint-disable-next-line backpack/use-components */
+      <img
+        alt={alt}
+        src={src}
+        {...imageRest}
```

Comment:
Shouldn't this come after all the other props?

Reply:
Another good catch, updating now!

Reply:
Done